_data/people/mweinberg2718.yml

```
@@ -6,3 +6,9 @@ institution: University of Chicago
 website:
 photo: /assets/images/team/Marc-Weinberg.jpg
 presentations:
+  - title: Delivery of columnar data to analysis systems
+    date: 2019-06-24
+    url: https://indico.cern.ch/event/765245/contributions/3470867/attachments/1869892/3076485/weinbergServiceX190626.pdf
+    meeting: ATLAS Software & Computing Week #62
```

```suggestion
    meeting: ATLAS Software & Computing Week \#62
```

I believe this is a comment in YAML.

---

It came out literally. For special characters, I usually just put the whole text in quotes. I'll fit it.

---

Actually, it came out literally for Brian Lin's version of this, which I put in quotes.